pennylane_lightning/core/src/simulators/lightning_kokkos/catalyst/CMakeLists.txt

```diff
@@ -73,19 +73,30 @@ else()
 
 endif()
 
+if(Kokkos_ENABLE_CUDA)
+    message(STATUS "Kokkos_ENABLE_CUDA is ON")
+
+    # KOKKOS_CUDA_OPTIONS:
+    # - disable_malloc_async: Disable asynchronous memory allocation to ensure
+    #   that memory operations complete before proceeding.
+    # - enable_lambda: Allow the use of lambda expressions in parallel
```

Comment:
Nice one! This flag was a blocker for using lambda functor when I was developing kernels for `lightning.kokkos`.

Reply:
would `enable_lambda` work for older nvcc version? Do we have to add nvcc version requirements as well? Same thing for `enable_constexpr`. `nvcc-7.5` and later?

Reply:
Good point! `enable_lambda` is supported for NVCC 7+ and `enable_constexpr` for 7.5+. I'll update the code to address this. Thanks!